### PR TITLE
Add pipeline for creating ACRs

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

**Review by timwebster9 — APPROVED** [review]

File: azure-pipelines.yaml [hunk 1]
```diff
@@ -3,12 +3,97 @@
...
+  - stage: CI
+    jobs:
+      - job: Validate
+        pool:
+          vmImage: 'Ubuntu-16.04'
+        steps:
+          - task: PublishPipelineArtifact@0
+            inputs:
+              artifactName: 'drop'
+              targetPath: '$(System.DefaultWorkingDirectory)/template.json'
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml:17]: This is i think the simplest azure devops has ever seen :)

File: azure-pipelines.yaml [hunk 2]
```diff
@@ -3,12 +3,97 @@
...
+                    overrideParameters: '-name hmctspublic'
+      - deployment: hmctsprivate
+        pool:
+          vmImage: 'Ubuntu-16.04'
+        environment: 'prod'
+        strategy:
+          # default deployment strategy
+          runOnce:
+            deploy:
+              steps:
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml:83]: May be create a new step template and reuse ? But this might never know, may not be worth it .

**Review by adusumillipraveen — APPROVED** [review]

**Comment by adusumillipraveen** [comment]
Lol :D

File: azure-pipelines.yaml (hunk 2 above)
Comment (timja) [reply azure-pipelines.yaml:83]: cbf, this will likely never even run again. waste of time even building this imo 😉

**Pull request merged**